index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <nav>
+
+        <label class="logo">Gerald</label>
+        <u>
```

Comment:
- [ ] Please use the `a` tag instead of the `label` tag for these three reasons: `convention`, `semantics`, and `user experience(UX)`. [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. Check below :point_down: my suggestions :slightly_smiling_face: 

```suggestion
        <a href='#' class="logo">Gerald</a>
        <u>
```